IOIOLibCore/src/main/java/ioio/lib/util/IOIOLooperProvider.java

```diff
@@ -30,7 +30,7 @@
 	 * @return An implementation of {@link IOIOLooper}, or <code>null</code> to
 	 *         skip.
 	 */
-	public abstract IOIOLooper createIOIOLooper(String connectionType,
-			Object extra);
+    IOIOLooper createIOIOLooper(String connectionType,
```

Comment:
Marking this change for myself as it may imply some necessary changes to my own project. Need to test.

Reply:
If you find something, please open a pull request